pkg/cmd/issue/close/close.go

```diff
@@ -64,6 +67,31 @@ func closeRun(opts *CloseOptions) error {
 		return err
 	}
 
+	var baseRepo ghrepo.Interface
+	if _, r := shared.IssueMetadataFromURL(opts.SelectorArg); r != nil {
+		baseRepo = r
+	} else {
+		baseRepo, err = opts.BaseRepo()
+		if err != nil {
+			return err
+		}
+	}
```

Comment:
As the `opts.SelectorArg` can be a URL we need to see if a host is specified in it for use with the feature detector.

Reply:
Yeah, good point. Not sure how I missed that.